aspnetcore/security/key-vault-configuration.md

```diff
@@ -57,6 +57,48 @@ When you run the app, a webpage shows the loaded secret values:
 
 ![Browser window showing secret values loaded via the Azure Key Vault Configuration Provider](key-vault-configuration/_static/sample1.png)
 
+## Bind an array to a class
+
+The provider is capable of reading configuration values into an array for binding to a POCO array.
+
+When reading from a configuration source that allows keys to contain colon (`:`) separators, a numeric key segment is used to distinguish the keys that make up an array (`:0:`, `:1:`, … `:{n}:`). For more information, see [Configuration: Bind an array to a class](xref:fundamentals/configuration/index#bind-an-array-to-a-class).
+
+Azure Key Vault keys can't use a colon as a separator. The approach described in this topic uses double dashes (`--`) as a separator for hierarchical values (sections). Array keys are stored in Azure Key Vault with double dashes and numeric key segments (`--0--`, `--1--`, … `--{n}--`).
+
+Examine the following [Serilog](https://serilog.net/) JSON file configuration. There are two `WriteTo` sinks specified in a JSON array:
```

Comment:
Reword this 2nd sentence. Not sure what's meant by "sinks". Did you mean to say "There are two object literals defined within the `WriteTo` array:"?